### PR TITLE
Release 23.5.0

### PR DESCRIPTION
**Pull request by beeankha**
## Description

✂️ snip snip ✂️ the making of a new release.

Xref https://github.com/conda/conda/issues/12625

### CODE REVIEW

File: .authors.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - name: Michael Sarahan
   email: msarahan@continuum.io
-  num_commits: 665
+  num_commits: 1330
```
Comment (dholth) [inline .authors.yml:3]: unfortunately the unpatched rever multi-counts commits from aliased contributors.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,83 @@
 [//]: # (current developments)
 
+## 23.5.0 (2023-05-17)
+
+### Enhancements
+
+* Add `conda doctor` subcommand plugin. (#474)
+* Add Python 3.11 support. (#12256)
+* Added `conda list --reverse` to return a reversed list of installed packages. (#11954)
```
Comment (kathatherine) [inline CHANGELOG.md:9]:
```suggestion
* Add `conda list --reverse` to return a reversed list of installed packages. (#11954)
```

**Review by kathatherine — COMMENTED** [review]
One verb change for changelog consistency.

**Review by kathatherine — DISMISSED** [review]
LGTM!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,83 @@
...
+### Docs
+
+* Change the README example from IPython Notebook and NumPy to PyTorch. (#12579)
+* Discuss options available to properly configure mirrored channels. (#12583, #12641)
+* Add `flake8-docstrings` to `pre-commit`. (#12620)
+
+### Other
+
+* Update retry language in flexible solve and repodata logs to be less ominous. (#12007)
+* Improve repodata / subdir_data programming interface (#12521). Index cache metadata has changed to `.info.json` to better align with the [draft CEP](https://github.com/conda-incubator/ceps/pull/48). Improve cache locking when using `jlap`. Improve `jlap` logging. (#12572)
```
Comment (beeankha) [inline CHANGELOG.md:52]:
```suggestion
* Update retry language in flexible solve and `repodata` logs to be less ominous. (#12007)
* Improve `repodata` / `subdir_data` programming interface (#12521). Index cache metadata has changed to `.info.json` to better align with the [draft CEP](https://github.com/conda-incubator/ceps/pull/48). Improve cache locking when using `jlap`. Improve `jlap` logging. (#12572)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,83 @@
...
+* Format with `black` and replaced `pre-commit`'s `darker` hook with `black`. (#12554)
+* Format with `isort` and add `pre-commit` `isort` hook. (#12554)
+* Add functional tests around Conda's content trust code. (#11805)
```
Comment (beeankha) [inline CHANGELOG.md:55]:
```suggestion
* Add functional tests around conda's content trust code. (#11805)
```

**Review by kathatherine — DISMISSED** [review]

**Pull request merged**